torchtune/modules/peft/dora.py

```diff
@@ -104,11 +104,28 @@ def initialize_dora_magnitude(self):
         """
         DoRA initializes the magnitude vector such that its outputs are initially
         identical to standard LoRA's outputs.
+
+        This must be called after loading/initializing base model and LoRA params.
         """
+        if any(
+            [
+                self.weight.is_meta,
+                self.lora_a.weight.is_meta,
+                self.lora_b.weight.is_meta,
+            ]
+        ):
+            raise ValueError(
```

Comment:
cc @joecummings on our docstring linter being broken

Could you add a `Raise` in the docstring here please? @ebsmothers